src/index.js

```diff
@@ -1,5 +1,8 @@
 /* eslint no-console:0, global-require:0, import/no-dynamic-require:0 */
 /* eslint complexity: [1, 13] */
+// need trimStart and trimEnd for Node 8
+import 'core-js/modules/es.string.trim-start';
```

Comment:
Prettier 2 uses these functions but they are not supported by Node 8. Node 8 reached its end-of-life on December 31, 2019. Node 8 support could be sunset in the future.